exercises/ruby-1-b/.meta/config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "solution_files": ["ruby_1_b.rb"],
```

Comment:
Just to confirm. You still need to update this file, right? It is now the same as exercise A.

Reply:
Correct. Both exercises are identical. Purely there so we can all beta test on Ruby without it mattering which exercise we get assigned, basically.

Reply:
Okay cool!